### PR TITLE
Fixes #10568

### LINKED ISSUE #10568: Slimes aren't logged
Player slimes can kill people with no logging.

Example shift http://game.ss13.moe/logs/2016/06-June/15-Wednesday.log, slime (127) attacked several people resulting in deaths, spread baby slimes all over the station, and no logs. The biggest problem here isn't one player whose behavior is correctable, but it's the exploitable hole in our logging.

### PR DESCRIPTION
**Pull request by ghost**
fixes #10568

attack log example:

[16:09:35] Started to feed on Louis Wise (@drcelt)! = null
[16:09:50] Has glomped on Louis Wise , played by @drcelt! = null
[16:10:35] Started to feed on Louis Wise (@drcelt)! = null
[16:11:27] Evolved to an adult slime at x=290,y=202,z=1! = null
[16:11:31] Started to feed on Louis Wise (@drcelt)! = null
[16:12:04] Started to feed on Louis Wise (@drcelt)! = null
[16:12:21] Reproduced to make more slimes at x=290,y=202,z=1! = null

### CODE REVIEW

File: code/modules/mob/living/carbon/alien/larva/larva.dm [hunk 1]
```diff
@@ -141,6 +141,8 @@
 
 		for(var/mob/O in viewers(src, null))
 			visible_message("<span class='danger'>\The [M] glomps \the [src]!</span>")
+			attack_log += text("\[[time_stamp()]\] <font color='orange'>Has been glomped on by \the [M] [M.ckey ? ", played by [M.ckey]" : ""]!</font>")
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/alien/larva/larva.dm]: This is inside a loop of viewers and will generate one for each.

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -274,6 +274,8 @@
 		for(var/mob/O in viewers(src, null))
 			if ((O.client && !( O.blinded )))
 				O.show_message(text("<span class='danger'>The [M.name] glomps []!</span>", src), 1)
+				attack_log += text("\[[time_stamp()]\] <font color='orange'>Has been glomped on by \the [M] [M.ckey ? ", played by [M.ckey]" : ""]!</font>")
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/human/human.dm]: Ditto.

File: code/modules/mob/living/carbon/monkey/monkey.dm [hunk 3]
```diff
@@ -428,6 +428,8 @@
 		for(var/mob/O in viewers(src, null))
 			if ((O.client && !( O.blinded )))
 				O.show_message(text("<span class='danger'>The [M.name] glomps []!</span>", src), 1)
+				attack_log += text("\[[time_stamp()]\] <font color='orange'>Has been glomped on by \the [M] [M.ckey ? ", played by [M.ckey]" : ""]!</font>")
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/monkey/monkey.dm]: Ditto.

File: code/modules/mob/living/carbon/slime/slime.dm [hunk 4]
```diff
@@ -307,6 +307,8 @@
 		for(var/mob/O in viewers(src, null))
 			if ((O.client && !( O.blinded )))
 				O.show_message(text("<span class='danger'>The [M.name] has glomped []!</span>", src), 1)
+				attack_log += text("\[[time_stamp()]\] <font color='orange'>Has been glomped on by \the [M] [M.ckey ? ", played by [M.ckey]" : ""]!</font>")
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/slime/slime.dm]: Ditto.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 5]
```diff
@@ -1031,6 +1031,8 @@
 		for(var/mob/O in viewers(src, null))
 			if ((O.client && !( O.blinded )))
 				O.show_message(text("<span class='danger'>The [M.name] glomps []!</span>", src), 1)
+				attack_log += text("\[[time_stamp()]\] <font color='orange'>Has been glomped on by \the [M] [M.ckey ? ", played by [M.ckey]" : ""]!</font>")
```
Comment (ComicIronic) [inline code/modules/mob/living/silicon/robot/robot.dm]: Ditto.

**Comment by ComicIronic** [comment]
Use the add_logs() proc instead.

I suggest

`add_logs(slime, target, "glomped on", 0 /*this means no admin logs*/)`

This will cover both the attacking and being-attacked logs for the mobs you pass in.

File: code/modules/mob/living/carbon/alien/larva/larva.dm (hunk 1 above)
Comment (ghost) [inline code/modules/mob/living/carbon/alien/larva/larva.dm]: whoops.

File: code/modules/mob/living/carbon/human/human.dm (hunk 2 above)
Comment (ghost) [inline code/modules/mob/living/carbon/human/human.dm]: whoops

File: code/modules/mob/living/carbon/monkey/monkey.dm (hunk 3 above)
Comment (ghost) [inline code/modules/mob/living/carbon/monkey/monkey.dm]: whoops

File: code/modules/mob/living/carbon/slime/slime.dm (hunk 4 above)
Comment (ghost) [inline code/modules/mob/living/carbon/slime/slime.dm]: whoops

File: code/modules/mob/living/silicon/robot/robot.dm (hunk 5 above)
Comment (ghost) [inline code/modules/mob/living/silicon/robot/robot.dm]: whoops

**Comment by ghost** [comment]
@ComicIronic Will do

**Comment by ghost** [comment]
@ComicIronic there you go

File: code/modules/mob/living/carbon/slime/powers.dm [hunk 6]
```diff
@@ -53,7 +53,8 @@
 		icon_state = "[colour] adult slime eat"
 	else
 		icon_state = "[colour] baby slime eat"
-
+	Victim.attack_log += text("\[[time_stamp()]\] <font color='orange'>Has been fed on by \the [src] [ckey ? ", played by [ckey]" : ""]!</font>")
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/slime/powers.dm]: You should also be using add_logs here, if that wasn't clear.

Comment (ghost) [inline code/modules/mob/living/carbon/slime/powers.dm]: right-o.

**Comment by ghost** [comment]
@ComicIronic did again

**Pull request merged**